src/plugins/usage_collection/server/routes/stats.ts

```diff
@@ -61,13 +62,17 @@ export function registerStatsRoute({
   metrics: MetricsServiceSetup;
   overallStatus$: Observable<ServiceStatus>;
 }) {
-  const getUsage = async (callCluster: LegacyAPICaller): Promise<any> => {
-    const usage = await collectorSet.bulkFetchUsage(callCluster);
+  const getUsage = async (
+    callCluster: LegacyAPICaller,
+    esClient: ElasticsearchClient
+  ): Promise<any> => {
+    const usage = await collectorSet.bulkFetchUsage(callCluster, esClient);
     return collectorSet.toObject(usage);
   };
 
-  const getClusterUuid = async (callCluster: LegacyAPICaller): Promise<string> => {
-    const { cluster_uuid: uuid } = await callCluster('info', { filterPath: 'cluster_uuid' });
+  const getClusterUuid = async (esClient: ElasticsearchClient): Promise<string> => {
+    const result = await esClient.info<{ cluster_uuid: string }>();
+    const { cluster_uuid: uuid } = result.body;
```

Comment:
This client is causing the 401 tests to fail. What if we revert it for this PR and fix it in a follow-up?

Reply:
[Done](https://github.com/elastic/kibana/pull/78046/commits/12ca4616bdbb274cd6ca8ea21d6d15adb262da07)